test/example_data.dart

```diff
@@ -127,6 +127,7 @@ StreamMessage streamMessage({
   String? topic,
   String? content,
   String? contentMarkdown,
+  List<String>? flags,
```

Comment:
Since we're adding this feature to `eg.streamMessage`, let's add it to `eg.dmMessage` at the same time to keep them nicely parallel.